receiver/awsecscontainermetricsreceiver/doc.go

```diff
@@ -0,0 +1 @@
+package awsecscontainermetricsreceiver
```

Comment:
Should have copyright text and doc. e.g. https://github.com/open-telemetry/opentelemetry-collector/blob/8aa273184455591cad278c92c7cfcf75ad353d57/extension/pprofextension/doc.go#L18-L17